src/Mod/CppModule.cpp

```diff
@@ -0,0 +1,60 @@
+#define NOMINMAX
+
+#include <filesystem>
+
+#include <Mod/CppModule.hpp>
+
+namespace RC
+{
+    CppModule::CppModule(UE4SSProgram& program, std::wstring&& mod_name, std::wstring&& mod_path) : Mod(program, std::move(mod_name), std::move(mod_path))
+    {
+        m_dlls_path = m_mod_path + L"\\dlls";
+
+        if (!std::filesystem::exists(m_dlls_path))
+        {
+            set_installable(false);
```

Comment:
We should use `Output::send<LogLevel::Warning>(...)` here to let the user know why the dll couldn't be loaded.
In the future, this message could be sent via `set_installable` instead and outputted alongside the regular `Was unable to install mod` message.

Reply:
Oh I did not see that, I thought it was using the printf format because I saw it being used somewhere in `UE4SSProgram.cpp`